packages/consent/consent-wrapper-onetrust/README.md

```diff
@@ -1,64 +1,124 @@
+- Debugging: this library expects the [OneTrust Banner SDK](https://community.cookiepro.com/s/article/UUID-d8291f61-aa31-813a-ef16-3f6dec73d643?language=en_US) to be available in order interact with OneTrust. This library derives the group IDs that are active for the current user from the `window.OneTrustActiveGroups` object provided by the OneTrust SDK. [Read this for more information [community.cookiepro.com]](https://community.cookiepro.com/s/article/UUID-66bcaaf1-c7ca-5f32-6760-c75a1337c226?language=en_US).
 
-## Install dependency
+## For `npm` library users
+
+1. Install the package
 
 ```sh
+# npm
 npm install @segment/analytics-consent-wrapper-onetrust
 
-# or
-
+# yarn
 yarn add @segment/analytics-consent-wrapper-onetrust
-```
 
-## For `npm` library users
+# pnpm
+pnpm add @segment/analytics-consent-wrapper-onetrust
+```
 
-- Use the following initialization code
+2. Initialize alongside analytics
 
 ```ts
-import { oneTrust } from '@segment/analytics-consent-wrapper-onetrust'
+import { withOneTrust } from '@segment/analytics-consent-wrapper-onetrust'
 import { AnalyticsBrowser } from '@segment/analytics-next'
 
 export const analytics = new AnalyticsBrowser()
 
-oneTrust(analytics)
-analytics.load({ writeKey: '<MY_WRITE_KEY'> })
+withOneTrust(analytics).load({ writeKey: '<MY_WRITE_KEY'> })
 
 ```
 
-## For snippet users (window.analytics) who _also_ use a bundler like webpack
-### _NOTE:_ a pre-bundled version that can be loaded through a `<script>` is on the roadmap, but _not_ supported at this point
+## For snippet users (window.analytics)
 
-- Install the dependency (see directions)
-- Delete the `analytics.load()` line from the snippet
+1. In your head
 
-```diff
-- analytics.load("<MY_WRITE_KEY>");
+```html
+<head>
+  <!-- Add OneTrust -->
+  <script
+    src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js"
+    type="text/javascript"
+    charset="UTF-8"
+    data-domain-script="YOUR-DOMAIN-SCRIPT-ID"
+  ></script>
+
+  <!-- Add OneTrust Consent Wrapper -->
+  <script src="https://unpkg.com/@segment/analytics-consent-wrapper-onetrust@X.X.X/dist/umd/analytics-onetrust.umd.js"></script>
+
+  <!--
+    Add / Modify Segment Analytics Snippet
+    * Find and replace: analytics.load() -> withOneTrust(analytics).load()
+  -->
+  <script>
+    !function(){var analytics=window.analytics...
+    ....
+    withOneTrust(analytics).load() // replace analytics.load()
+  </script>
+</head>
 ```
 
-- Use the following initialization code
+#### ⚠️ Reminder: _delete_ `analytics.load('....')` from the original Segment snippet so that only .load() is only called after `withOneTrust` is called. See comment in example above.
```

Comment:
Oh, this is great to callout things, I'm going to borrow this for an RN readme change